app/assets/javascripts/discourse/app/components/relative-time-picker.js

```diff
@@ -35,17 +35,38 @@ export default Component.extend({
     this.hiddenIntervals = this.hiddenIntervals || [];
   },
 
+  _roundedDuration(duration) {
+    let rounded = parseFloat(duration.toFixed(2));
+
+    // showing 2.00 instead of just 2 in the input is weird
+    if (rounded % 1 === 0) {
+      return parseInt(rounded, 10);
+    }
```

Comment:
2.10 ? maybe we strip that zero as well ?

Reply:
Hmm good point, will do that

Reply:
Oh wait that's not actually a problem because I do `parseFloat`, so works as is.